### PR TITLE
Refactor: first demo for developing this SDK

### PR DESCRIPTION
**Pull request by XadillaX**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# rocketmq-client-nodejs
+# RocketMQ Client for Node.js
+
+It still under development.
```
Comment (vongosling) [inline README.md]: It would be helpful for community users to be involved in if we could make some comments on how to build from scratch.

Comment (XadillaX) [reply README.md]: This PR is for several community volunteers who discussed last weekend. We will work together to finish the first available version.

Comment (XadillaX) [reply README.md]: I've updated the README.md

**Review by wlliqipeng — COMMENTED** [review]
LGTM

**Pull request merged**